creatorkit/Toolbox.md

```diff
@@ -25,17 +25,15 @@ There are also 2 ways to power Clicker boards
 
 You can get pre-built releases or source code of the Creator Kit projects to work with. The Releases will allow you to set up a working system quickly without having to build any code. The source code will allow you to inspect/extend the projects and build the releases yourself.
 
-[Releases](http://github.com/CreatorKit)
+[Releases](http://github.com/CreatorKit/build)
 
-[Source Code](http://github.com/CreatorKit/manifest)
-
-See [building code from source](#building-code-from-source) for information on how to use the source code.
+See [building from source](#building-from-source) for information on how to use the source code.
 
 ### Connecting to Ci40 via Serial
 
 ![serial](../images/ci40serial.jpg)
 
-The next step is to connect to your Ci40 using a serial terminal. Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like set to the standard 115200-N-8-1. The following is an example using miniterm in Ubuntu
+Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like set to the standard 115200-N-8-1. The following is an example using miniterm in Ubuntu
```

Comment:
I'd suggest using microUSB to USB Type-A cable.
Correction: You can use any serial terminal program you like to set the standard 115200-N-8-1.  The following command is an example using miniterm.py on Ubuntu